js/query-parser.js

```diff
@@ -38,6 +38,21 @@ Fliplet.Registry.set('dynamicListQueryParser', function() {
   });
   this.queryPreFilter = _(this.pvPreFilterQuery).size() > 0;
 
+  // We can sort only by one column that is why this syntax doesn't support
+  // ?dynamicListFilterColumn=Name,Age&dynamicListFilterOrder=asc
```

Comment:
There was a mistake in the spec. The query parameters should refer to `sort` not `filter`.

```suggestion
  // ?dynamicListSortColumn=Name,Age&dynamicListSortOrder=asc
```